cmd/gardenlet/app/bootstrappers/seed_config.go

```diff
@@ -69,6 +69,10 @@ func (s *SeedConfigChecker) Start(ctx context.Context) error {
 	return nil
 }
 
+func isSameIPFamily(ip net.IP, ipNet *net.IPNet) bool {
+	return (ip.To4() == nil && ipNet.IP.To4() == nil) || (ip.To4() != nil && ipNet.IP.To4() != nil)
+}
```

Comment:
This function is very similar to the (unexported) function `matchAddrFamily(...)` in the standard library, but it is also different. Maybe you want to adapt your code:
https://cs.opensource.google/go/go/+/master:src/net/ip.go;drc=2e71489ed1a30f95d0e11685536f5d61d9e858b3;l=373

Reply:
Done.